src/oop/user_defined_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,28 @@
+# oop-user_defined_classes-medium-q3
+
+## Java
+
+Teaching young children to make animal sounds is fun and helps them 
+improve their language skills. Make a Java class called Horse
+that has an attribute called weight, a constructor that 
+initializes the weight, and one method called makeSound 
+that returns "Neigh!!".
+
+
+
+### Horse Class
+
+The `Horse` class you are given has the following items:
+
+- A `Horse` class name
+- An empty `Horse` constructor 
+- A `double` called **weight**
+
+You must add the following items to the class:
+
+- Constructors:
+    - Initializes **weight**
```

Comment:
I think for this, we add that they need to check that the weight value is not negative. If it is, they can initialize it to zero instead.